src/wallet/rpc/addresses.cpp

```diff
@@ -786,4 +786,48 @@ RPCHelpMan walletdisplayaddress()
     };
 }
 #endif // ENABLE_EXTERNAL_SIGNER
+
+RPCHelpMan getxpub()
```

Comment:
Maybe `getxupub` and `listdescriptors` can be combined into a single command?

Reply:
I think it's useful to have them be separate.